### PR TITLE
MC-15611: doc for create and update dns record

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made
-Doc for create dns record
-Doc for update dns record

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/stackpath/_dns_records.md [hunk 1]
```diff
@@ -122,6 +122,100 @@ Attributes | &nbsp;
...
+---- | -----------
+`zoneId`<br/>*UUID* | The ID of the zone for which we want to delete the record. This parameter is required.
+
+Attributes | &nbsp;
+------- | -----------
+`name`<br/>*string* | The name of the record. Use the value "@" to denote current root domain name.
+`type`<br/>*string* | A DNS record's type. DNS record types describe the record's behavior.
+`ttl` <br/>*int* | A DNS record's time to live. A record's TTL is the number of seconds that the record should be cached by DNS resolvers. Use lower TTL values if you expect zone records to change often. Use higher TTL values for records that won't change to prevent extra DNS lookups by clients.
+`data`<br/>*string* | A DNS record's value.
+`weight` <br/>*int* | A DNS record's priority. A resource record is replicated in StackPath's DNS infrastructure the number of times of the record's weight, giving it a more likely response to queries if a zone has records with the same name and type.
```
Comment (fyrnaga) [inline source/includes/stackpath/_dns_records.md:173]: should mark it as optional in a separate table:
```
required
-----

optional
-----
```

File: source/includes/stackpath/_dns_records.md [hunk 2]
```diff
@@ -122,6 +122,100 @@ Attributes | &nbsp;
...
+
+<!-------------------- UPDATE A DNS RECORD -------------------->
+
+#### Update a DNS record
+
+```shell
+curl -X POST \
```
Comment (fyrnaga) [inline source/includes/stackpath/_dns_records.md]: this is a put

File: source/includes/stackpath/_dns_records.md [hunk 3]
```diff
@@ -122,6 +122,100 @@ Attributes | &nbsp;
...
+`zoneId`<br/>*UUID* | The ID of the zone for which we want to delete the record. This parameter is required.
+
+Attributes | &nbsp;
+------- | -----------
+`name`<br/>*string* | The name of the record. Use the value "@" to denote current root domain name.
+`type`<br/>*string* | A DNS record's type. DNS record types describe the record's behavior.
+`ttl` <br/>*int* | A DNS record's time to live. A record's TTL is the number of seconds that the record should be cached by DNS resolvers. Use lower TTL values if you expect zone records to change often. Use higher TTL values for records that won't change to prevent extra DNS lookups by clients.
+`data`<br/>*string* | A DNS record's value.
+`weight` <br/>*int* | A DNS record's priority. A resource record is replicated in StackPath's DNS infrastructure the number of times of the record's weight, giving it a more likely response to queries if a zone has records with the same name and type.
+
```
Comment (fyrnaga) [inline source/includes/stackpath/_dns_records.md]: same comment than above

File: source/includes/stackpath/_dns_records.md [hunk 4]
```diff
@@ -122,6 +122,100 @@ Attributes | &nbsp;
...
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId&operation=edit</code>
```
Comment (fyrnaga) [inline source/includes/stackpath/_dns_records.md]: We should not have to set the edit in the operation. It should be the default
```suggestion
<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords/:id?zoneId=:zoneId</code>
```

Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md]: 19182ad

File: source/includes/stackpath/_dns_records.md (hunk 3 above)
Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md]: 19182ad

File: source/includes/stackpath/_dns_records.md (hunk 2 above)
Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md]: 19182ad

File: source/includes/stackpath/_dns_records.md (hunk 1 above)
Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md:173]: 19182ad

File: source/includes/stackpath/_dns_records.md [hunk 5]
```diff
@@ -122,6 +122,106 @@ Attributes | &nbsp;
...
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId</code>
```
Comment (fyrnaga) [inline source/includes/stackpath/_dns_records.md]:
```suggestion
<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords/:id?zoneId=:zoneId</code>
```

**Review by fyrnaga — APPROVED** [review]
lgtm

**Review by HassanHaidar — APPROVED** [review]
Looks good 👍

File: source/includes/stackpath/_dns_records.md [hunk 6]
```diff
@@ -122,6 +122,106 @@ Attributes | &nbsp;
...
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId</code>
+
+Create a DNS record in a given [environment](#administration-environments).
+
+Query Params | &nbsp;
+---- | -----------
+`zoneId`<br/>*UUID* | The ID of the zone for which we want to delete the record. This parameter is required.
```
Comment (GuillaumeVialle) [inline source/includes/stackpath/_dns_records.md]: delete -> create

File: source/includes/stackpath/_dns_records.md [hunk 7]
```diff
@@ -122,6 +122,106 @@ Attributes | &nbsp;
...
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords/:id?zoneId=:zoneId</code>
+
+Update a DNS record in a given [environment](#administration-environments).
+
+Query Params | &nbsp;
+---- | -----------
+`zoneId`<br/>*UUID* | The ID of the zone for which we want to delete the record. This parameter is required.
```
Comment (GuillaumeVialle) [inline source/includes/stackpath/_dns_records.md]: update the record

File: source/includes/stackpath/_dns_records.md (hunk 6 above)
Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md]: 1fc6163

File: source/includes/stackpath/_dns_records.md (hunk 7 above)
Comment (AnKhangC) [reply source/includes/stackpath/_dns_records.md]: 1fc6163

**Pull request merged**